style.css

```diff
@@ -0,0 +1,735 @@
+    height: 5vh;
+  }
+
+  .navbar-items {
+    height: 5vh;
+    justify-content: space-between;
+    margin-left: -17%;
+  }
+
+  .nawbar-btn {
+    font-size: 75%;
+    margin: 1% 5% 0 5%;
+  }
+
+  .navbar-items {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .navbar-items #nav-img {
+    order: 1;
+    height: 4.1vh;
+    margin: 0 1% 0 -1%;
+  }
+
+  .navbar-items #nav-button-1 {
+    order: 2;
+  }
+  .navbar-items #nav-button-2 {
+    order: 2;
+  }
+
+  .navbar-items #nav-button-3 {
+    order: 3;
+  }
+
+  .navbar-items #nav-button-4 {
+    order: 3;
+  }
+
+  #nav-text {
+    visibility: hidden;
+  }
+
+  .nav-btn {
+    width: 5vw;
+    height: 5vh;
+  }
+
+  header {
+    grid-row: span 1;
+    grid-column: span 12;
+    width: 100%;
+    height: 100vw;
+  }
+
+  .big-news {
+    grid-row: span 1;
+    grid-column: span 12;
+    width: 100vw;
+    height: 100%;
+    padding: 0% 2% 0% 2%;
+    margin-top: -8%;
+  }
+
+  #bignewz-first-h1 {
+    font-size: 2.5rem;
+    margin: 8% 0% 1% 0%;
+  }
+
+  .big-news h1 {
+    margin: 1% 0% 1% 0%;
+    font-size: 100%;
+  }
+
+  .big-news p {
+    margin: 0% 1% 1% 0%;
+
+    font-size: 75%;
+  }
+
+  .stream-big-btn {
+    width: 50vw;
+    margin: 3% 1% 0% 0%;
+    font-size: 0.8rem;
+  }
+  .regular-news-cards {
+    grid-row: span 1;
+    grid-column: span 12;
+  }
+
+  #footer-text {
+    font-size: 0.1%;
+    padding-top: 2%;
+  }
+}
+
+@media (max-width: 608px) {
+  .navbar-items {
+    margin-left: -18.5%;
+  }
+}
+
+@media (max-width: 550px) {
+  .navbar-items {
+    margin-left: -20%;
+  }
+  .nawbar-btn {
+    font-size: 75%;
+    margin: 1% 4% 0 5%;
+  }
+}
+@media (max-width: 500px) {
+  .navbar-items {
+    margin-left: -22%;
+  }
+  .nawbar-btn {
+    font-size: 75%;
+    margin: 1% 4% 0 4%;
+  }
+}
+
+@media (max-width: 450px) {
+  .navbar-items {
+    margin-left: -25%;
+  }
+  .nawbar-btn {
+    font-size: 75%;
+    margin: 1% 3% 0 3%;
+  }
+}
+
+@media (max-width: 400px) {
+  .navbar-items {
+    margin-left: -28%;
+  }
+  .nawbar-btn {
+    font-size: 75%;
+    margin: 1% 2% 0 3%;
+  }
+}
+
+@media (max-width: 375px) {
+  .container-nav {
+    justify-content: center;
+    align-content: center;
+    margin: -1% 0 0 10%;
+  }
+
+  .navbar-items {
+    margin-left: -35%;
+  }
+
+  .navbar-items #nav-img {
+    margin-left: -2%;
+    padding: 0 1% 0 0;
+    height: 3vh;
+  }
+
+  .nawbar-btn {
+    font-size: 75%;
+    margin: 0% 1.4% 0 1.5%;
+  }
+
+  .regular-news-cards {
+    margin-top: 4%;
+  }
+  .regular-news-cards img {
+    height: 200px;
+  }
+  .footer-div {
+    width: 115%;
+  }
+  #footer-text {
+    font-size: 5px;
+  }
+}
+
+@media (max-width: 359px) {
+  .navbar-items {
+    margin-left: -40%;
+  }
+  .nawbar-btn {
+    font-size: 75%;
+    margin: 1% 2% 0 2%;
+  }
+}
+
+@media (max-width: 355px) {
+  .navbar-items {
+    margin-left: -40%;
+  }
+  .nawbar-btn {
+    font-size: 75%;
+    margin: 1% 2% 0 2%;
+  }
+
```

Comment:
How come you used 10 media queries?

Reply:
Hi Kaliine, I did change my header and "container-regular-footer" but cant push the changes :/
And the media queries is for the navbar to always be in the right spot (next time when I use the position:fix" I'm going to read what this type of position doing to the element, Like 65% of my work was to fix this navbar, haha, and if I would to this task again I would do a other soulotion - ten media queries is just for to fix the navbar!

Thanks for good and constructive feedback! :)